sopel/bot.py

```diff
@@ -326,12 +340,13 @@ def remove_plugin(self, plugin, callables, jobs, shutdowns, urls):
                     regex = func.url_regex
                     if func == self.memory['url_callbacks'].get(regex):
                         self.unregister_url_callback(regex)
+                        LOGGER.debug('URL Callback unregistered %r', regex)
         except:  # noqa
-            # TODO: consider logging?
-            raise  # re-raised
+            raise
```

Comment:
Why remove this comment?

Also, why not log the exception that happened? It could be useful information if the plugin machinery ever catastrophically fails in the wild.

Reply:
If the plugin machinery catastrophically fails in the wild, then a higher level of the bot will handle the error and log it anyway.

By the way, this try/except/else doesn't have a purpose anymore. We could just let the code fail without enclosing it into that, since we don't do anything in the `except`, and we don't rely on a `finally` clause.

Reply:
Hmm, might as well remove it then.

Reply:
Done.